.github/workflows/lint.yaml

```
@@ -1,4 +1,4 @@
-name: CloudFormation
+name: CloudFormation lint
```

```suggestion
name: Lint
```

This was questionable naming on my part, the CFN part is self-evident from the repo name so I'd remove it to keep the check names concise.